device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -170,7 +170,7 @@ public void onMessageSent(Message message, Throwable e)
                 //Codes_SRS_IOTHUBTRANSPORT_34_005: [If there was a packet in the inProgressPackets queue tied to the
                 // provided message, and the provided throwable is null, this function shall set the status of that
                 // packet to OK_EMPTY and add it to the callbacks queue.]
-                this.log.trace("Message was sent by this client, adding it to callbacks queue with OK_EMPTY ({})", message);
+                IotHubTransport.log.trace("Message was sent by this client, adding it to callbacks queue with OK_EMPTY ({})", message);
```

Comment:
I'm not sure I understand why we are making this particular change. Does the related warning say why to call into the log statically rather than through ```this```?

Reply:
I also thought about this one in particular if it is needed or not. I do not know answer to whether this is solving a warning, not sure how to figure that out other than running code cleanup twice. Are you concerned for readability here?

Reply:
yes it resolves this warning:
![image](https://user-images.githubusercontent.com/41314966/103717517-89b6e000-4f7a-11eb-918a-71a435ded0bb.png)

Reply:
removed this instead.